OxfordBodleian/Juel-Jensen/BDLaethe35.xml

```diff
@@ -227,6 +751,157 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                           </decoNote>
 
                          </decoDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#1r"/>
+                                    <desc type="Unclear">Note in Geez recording a familial or symbolic kinship.</desc>
+                                    <q xml:lang="gez">
```

Comment:
Technically it would be nice to have person records for the two persons, or maybe at least mention their names in the English summary of the note

Reply:
They are really the equivalent of names just scribbled on an empty leaf. I added their names in the summary :)